experimental/sdk/webpubsub-socketio-extension/src/SIO/index.ts

```diff
@@ -49,4 +76,33 @@ export async function useAzureSocketIO(
   return this;
 }
 
+function getDefaultNegotiateHandler(
+  engine: WebPubSubEioServer
+): (
+  req: IncomingMessage,
+  res: ServerResponse,
+  getClientAccessTokenUrl: (options?: GenerateClientTokenOptions) => Promise<string>
+) => Promise<void> {
+  return async (
+    req: IncomingMessage,
+    res: ServerResponse,
+    getClientAccessTokenUrl: (options?: GenerateClientTokenOptions) => Promise<string>
+  ): Promise<void> => {
+    let statusCode = 400,
+      message = "Bad Request";
+    try {
+      const username = parse(req.url || "", true).query["username"] as string;
+      const endpointWithToken = await getClientAccessTokenUrl({ userId: username ?? "" });
+      statusCode = 200;
+      message = endpointWithToken;
+    } catch (e) {
+      statusCode = 400;
```

Comment:
not all the exceptions are 400, 401/500 are also possible

Reply:
Now it will return 500 if
1. `serviceClient.getClientAccessToken` throw exception
2. `getGenerateClientTokenOptions` (provided by user)throw exceptions. 

If this correct?